tests/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import {render, screen} from '@testing-library/react'
+import Index from '../pages/index'
+
+describe("Exemplar-react-test renderer Snapshot test", () => {
```

Comment:
what's Exemplar-react-test from? googled and didn't find it

Reply:
It's me messing up the dashes in Exemplar react-test-renderer